### DISCUSSION TITLE
storybook-addon-a11y for react-native Storybook?

### DISCUSSION THREAD

**Discussion by ludovicobesana** [storybookjs/storybook#29321] [Help]
### Summary

Hi Storybook team!
I'm currently working on a React Native project and would like to integrate the storybook-addon-a11y to ensure accessibility standards across my components. However, I've noticed that the addon doesn't seem to support React Native Storybook as seamlessly as it does with the web version. Is there any plan to extend support for React Native?

Thanks for considering this, and I’m happy to help test or contribute where needed!

**Comment by dannyhw** [comment]
Its not currently available for native because it would mostly be a reimplementation and I'm not sure how to approach it yet, however I would love to see this become a thing. 

Do you know of testing tools for native that support detecting a11y issues?
